### PR TITLE
New: chinese simplified language package

### PR DESCRIPTION
**Pull request by chensixian**
i had completed the Chninese simplified package.

### CODE REVIEW

File: assets/js/langpack/zh_CN.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    "All Rights Reserved.",
+
+    "Find out more":
+    "详细了解",
+
+    "Back to main page":
+    "返回首页",
+
+    "Change language":
+    "Change language"
```
Comment (cyntss) [inline assets/js/langpack/zh_CN.json]: This needs translation

File: assets/js/langpack/zh_CN.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    "部落格",
+
+    "Press":
+    "回页首",
+
+    "Sponsored by:":
+    "向赞助方致以谢意︰",
+
+    "All Rights Reserved.":
+    "All Rights Reserved.",
```
Comment (cyntss) [inline assets/js/langpack/zh_CN.json]: Needs translation

File: assets/js/langpack/zh_CN.json [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    "Linux 映像创建工具。可以为实体机、虚拟机、Docker 创建部署用映像。Kiwi 是 SUSE Studio 的支柱。",
+
+    "Check all openSUSE tools":
+    "了解所有的 openSUSE tools",
+
+    "Our build tool, building all of our packages as well as ones for SUSE Linux Enterprise, Arch, Debian, Fedora, Scientific Linux, RHEL, CentOS, Ubuntu, and more.":
+    "我们的 build 工具。我们所有的包皆是用它建制。OBS 亦良好支持多种其它发布，像 SUSE Linux Enterprise、Arch、Debian、Fedora、Scientific Linux、RHEL、CentOS、Ubuntu 等。",
+
+    "Machinery supports configuration discovery, system validation, and service migration.":
+    "Machinery 用于 configuration discovery、system validation 和 service migration。",
```
Comment (cyntss) [inline assets/js/langpack/zh_CN.json]: Is this translation ok?

File: assets/js/langpack/zh_CN.json [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+    "Machinery is based on the idea of an universal system description.It is transparent,extensible, and crafted beautifully.":
+    "Machinery 的运作基于一个通用、简明、可扩展、精巧的 system description。",
```
Comment (cyntss) [inline assets/js/langpack/zh_CN.json]: can "system description" be translated?

**Comment by cyntss** [comment]
@chensixian thanks a lot for this translation. We have a chinese translation already, this is different?

Regardless of that, the translation looks good, there are a couple of strings that might need a second look. Apart from that, I kindly ask you to merge the new master, there where some changes in the string of the translation Template, you can have a look at the changes in the PR:

https://github.com/cyntss/opensuse-landing-page/pull/20

https://github.com/cyntss/opensuse-landing-page/pull/18

Let me know when its ready and I will have a look again :)

**Comment by chensixian** [comment]
to resive.
----
>We have a chinese translation already, this is different?

Chinese have two standard character sets: traditional Chinese (zh_TW), simplified Chinese (zh_CN).
The traditional Chinese is used in Taiwan, Hong Kong and Macau.
The simplified Chinese is used in mainland China.

**Pull request closed without merging**